sale_order_line_menu/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Sale Order Line Menu",
-    "version": "16.0.1.0.0",
+    "version": "16.0.1.0.1",
```

Comment:
you don't need to bump the version manually, the person that merges with the bot will pick what version to bump